test/controllers/password_reset_controller_test.exs

```diff
@@ -2,31 +2,33 @@ defmodule CodeCorps.PasswordResetControllerTest do
   @moduledoc false
 
   use CodeCorps.ApiCase, resource_name: :password_reset
-  alias CodeCorps.AuthToken
+  alias CodeCorps.{User, AuthToken}
 
-  @tag :authenticated
-  test "creates and renders resource when data is valid", %{conn: conn, current_user: current_user} do
+  test "creates and renders resource when data is valid", %{conn: conn} do
```

Comment:
I'd make this test name less generic. 

"updates user password when data is valid"